### ISSUE TITLE
offset from createPopper - Cannot read properties of undefined (reading 'x')

### ISSUE THREAD

**Issue by hanorine** [react-bootstrap/react-bootstrap#6685]
### Prerequisites

- [X] I am using the [correct version](https://github.com/react-bootstrap/react-bootstrap#bootstrap-compatibility) of React-Bootstrap for my version of Bootstrap
- [X] I have [searched](https://github.com/react-bootstrap/react-bootstrap/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/react-bootstrap/react-bootstrap/blob/master/CONTRIBUTING.md)

### Describe the bug

Greetings to the react-bootstrap team and thank you for your continuous development of this wonderful toolbox of react components.

I came across this issue multiple times in the past few days regarding the Popper component that you expose. 

```js
import { Popover} from 'react-bootstrap';
```

I am receiving the following error message:
```
offset.js:36 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'x')
    at offset (offset.js:36:1)
    at Object.forceUpdate (createPopper.js:128:1)
    at createPopper.js:141:1
    at nrWrapper (newrelic.js:1515:33)
    at new Promise (<anonymous>)
    at new r (newrelic.js:481:19)
    at createPopper.js:140:1
    at debounce.js:8:1
    at nrWrapper (newrelic.js:1515:33)
```

I created a minimal codesandbox example for your convenience [here](https://codesandbox.io/s/react-bootstrap-2-8-0-error-p2ywg6?file=/src/App.js). 

### Expected behavior

No errors are expected.

### To Reproduce

Using the codesanbox example [here](https://codesandbox.io/s/react-bootstrap-2-8-0-error-p2ywg6?file=/src/App.js), once loaded, hover over the Star Rating component. 

### Reproducible Example

https://codesandbox.io/s/react-bootstrap-2-8-0-error-p2ywg6?file=/src/App.js

### Screenshots

![image](https://github.com/react-bootstrap/react-bootstrap/assets/18309230/57a921e8-16bb-43b9-8fc6-06f69bf3b332)

![image](https://github.com/react-bootstrap/react-bootstrap/assets/18309230/eb5a78b9-5f14-493a-b711-b55828402a38)

### What version of React-Bootstrap are you using?

2.8.0

### What version of Bootstrap are you using?

5.3.1

**Comment by bnolan001** [comment]
I'm also seeing this when using the popper element too.  I'm using these versions:
* Bootstrap:  5.2.3
* React-Bootstrap:  2.7.0

**Comment by evanwjackson** [comment]
I'm seeing it too! These versions as well:
Bootstrap: 5.2.3
React-Bootstrap: 2.7.0

**Comment by psivaramakrishnan17** [comment]
I'm seeing the same output as detailed above. Below versions are being used:
Bootstrap: 5.2.3
React-Bootstrap : 2.7.0

Thanks.

**Comment by hanorine** [comment]
I figured out the issue and have updated the codesandbox to reflect it. Apparently a previous developer hadn't refactored correctly the usage of the OverlayTrigger and was passing in a value of `button` into the `placement` prop. Correcting it to `bottom` resolved the issue. Added typsecript to the file also highlighted the issue.

I would suggest that there be some error warning in case a value passed into the placement prop is not supported. This would help a lot as the error that I was getting was not clear where it exactly was coming from.
